AndelaOSP/art-dashboard
# Feature: Admin can Unassign assets

## What does this PR do?
It enables the admin un-assign an assets

## Description of Task to be completed?
      Given An existing user who has been assigned an asset
      When I unassigns that asset
      And I confirm the action
      Then the asset should be unassigned from that user
## How should this be manually tested?
 - Pull this branch then select an asset from the asset lists sub menu
 - Scroll down to the `Description` tab and click on the `Unassign` Button
   underneath the email of the user the asset is currently assigned to
-  Observe that the asset is unassigned
## What are the relevant pivotal tracker stories?
[#159022597](https://www.pivotaltracker.com/story/show/159022597)
## Any background context you want to add?
n/a
## Important notes
n/a
## Packages installed
n/a
## Deployment note
n/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
- [x ] Test
## Screenshots
n/a
